Sage/carbon
# Upgrade to Carbon Factory V1

Set ESLint threshold to 231

QA Instructions:

DEV QA Only:

Sanity Check app runs.
Run `gulp`, `gulp test`, `gulp test -b chrome`, `gulp test --coverage`, `gulp test --build`
